pages/upgrade-guide-2.x.md

```diff
@@ -196,6 +196,32 @@ QUnit.module( "router", {
 ```
 
 
+## Replace `assert.throws( block, string, message )` with `assert.throws( block, regexp, message )`
+
+The overload of `assert.throws()` which expected a block, error string, and assertion message has been removed and will now throw an exception. Use a regular expression instead.
```

Comment:
Sorry, that was my fault. A Chrome extension for navigating files within GitHub PRs ended up overlapping part of the diff, the one with "now throw" on the right.